src/chapters/setup-intro-data-types/naming-conventions.rst

```diff
@@ -0,0 +1,53 @@
+.. _naming-conventions:
+
+Java Naming conventions
+=======================
+
+Java has some very straightforward naming conventions. These are
+universally used by Java programmers, and differ in some cases from
+conventions commonly used in other languages.
+
+Again, these are conventions. Not following them will not prevent your
```

Comment:
Double nots might not be not easy to not follow.

How about something like, "Ignoring them will not prevent your code from running, as long as you are following Java’s naming rules. However, the conventions are what other Java coders expect to see."

Or we could go with a quote from Pirates of the Caribbean, "The code is more what you’d call ‘guidelines’ than actual rules.”

Reply:
changed this to use a little bit of both your suggestions